blueprints/hooks/light/light.yaml

```
@@ -351,20 +376,20 @@ action:
                       while: '{{ true }}'
                       sequence:
                         - choose:
-                            - conditions: '{{ state_attr(light,"brightness") == min_brightness }}'
+                            - conditions: '{{ state_attr(light,"brightness") == min_brightness or state_attr(light,"brightness") == none }}'
                               sequence:
                                 - service: light.turn_off
                                   data:
-                                    transition: 0.25
+                                    transition: '{{ light_transition / 1000 }}'
                                     entity_id: !input light
                           default:
                             - service: light.turn_on
                               data:
                                 brightness: '{{ [ [state_attr(light,"brightness")-step_long, min_brightness] | max, max_brightness] | min }}'
                                 transition: 0.25
                                 entity_id: !input light
-                        - delay:
-                            milliseconds: 250
+                            - delay:
+                                milliseconds: 250
```

```suggestion
                        - delay:
                            milliseconds: 250
```

I think this delay should not be indented: it should be applied as the last step in the loop body.